target/ppc/translate/fp-impl.c.inc

```diff
@@ -1493,6 +1493,29 @@ static bool trans_PLFS(DisasContext *ctx, arg_PLFS *a)
     return true;
 }
 
+static bool trans_PLFD(DisasContext *ctx, arg_PLS_D *a)
+{
+    TCGv ea;
+    TCGv_i64 t0;
+    arg_D d;
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_FPU(ctx);
+
+    if(!resolve_PLS_D(ctx, &d, a)){
+        return true;
+    }
+    ea = tcg_temp_new();
+    t0 = tcg_temp_new_i64();
+    do_ea_calc(ctx, d.ra, tcg_constant_tl(d.si), ea);
+    tcg_gen_qemu_ld_i64(t0, ea, ctx->mem_idx,
+                       MO_Q | ctx->default_tcg_memop_mask);
+    set_fpr(a->rt, t0);
+
+    tcg_temp_free_i64(t0);
+    tcg_temp_free(ea);
+    return true;
```

Comment:
I'd like to unite this function with the do_lstfs function, but we can do that in a later clean up

Reply:
At this point, do_lstfs isn't in the tree, so it'd need to be a future cleanup. 

The problem with that is that do_lstfs uses a function that specifically loads values as a single precision float, while PLFD loads as a double precision value. 
So we'd need to have a "do_lstfd" instead, but since there's no need to set a different access mode or using any very specific functions, I don't think its worth it